src/pages/components/progress-indicator/usage.mdx

```diff
@@ -45,41 +48,208 @@ set of steps, guiding toward the completion of a specified process.
-will accomplish within each step. Keep labels between one to two words.
+1. **Status indicator:** Communicates if a step is completed, incomplete, not
+   started, or has an error. See the
+   [status indicator pattern](/patterns/status-indicator-pattern/) for guidance.
+2. **Active step line:** Indicates completed steps and the step the user is
+   currently on.
+3. **Label:** Communicates what the user will accomplish in each step. Numbering
+   each step also makes the progression more obvious.
+4. **Inactive step line:** Indicates incomplete steps as well as steps that are
+   in an error, disabled, or skeleton state.
+5. **Optional label or error label:** Shows that a step is optional or in an
+   error state.
 
-#### Validation
+### Alignment
 
-Use validation to confirm that a previous step has been completed. If the user
-cannot proceed onto another step without first completing a task, use an
-[Inline Notification](/components/notification/code#inline-notification) to
-inform them.
+Progress indicators can be in vertical or horizontal position depending on the
+use case and the structure of the UI. When possible, arrange the progress
+indicator vertically for easier reading.
 
 <Row>
 <Column colLg={8}>
 
-![Example of the progress indicator in a payment UI](images/progress-indicator-usage-1.png)
+![Example of a progress indicator used with vertical alignment](images/progress-indicator-usage-4.png)
 
 </Column>
 </Row>
 
+### Placement
+
+The progress indicator component is often used in
+[forms](https://carbondesignsystem.com/components/form/usage). Forms can be
+placed on a full page, in a modal, or in a side panel.
+
+IMAGE TK: Get high res image. Likely needs design work first.
+
+<Row>
+<Column colLg={8}>
+
+![Examples of vertical and horizontal progress indicators in products](images/progress-indicator-usage-3.png)
+
+</Column>
+</Row>
+
+Checkboxes in a form should be placed at least 32px (layout-03) below or before
+the next component. Spacing of 24px (layout-02) or 16px (layout-01) can also be
+used when space is more restricted or if the form is more complex.
+
+For more information on spacing in forms, see our
+[form style guidance](https://carbondesignsystem.com/components/form/style/).
+
+## Content
+
+### Main elements
+
+#### Label
+
+The label should communicate what the user will accomplish in each step in one
+or two words. Choose a label that either starts with a verb ("Define endpoints")
+or is universally understood ("Shipping"), and avoid vague terms ("Processing").
+You may also number each step to make the progression more obvious.
+
+#### Optional label
+
+Optional labels indicate a step as optional or additional context about a step.
+Optional label is required when there step is invalid or in the error state.
+
+### Overflow content
+
+#### Label
+
+When there is not enough space, consider rewording the label or truncate the
+label text with an ellipsis and provide a tooltip to convey additional
```

Comment:
It may be good to show an image of what this would look like.

Reply:
Image added!